src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final OrderResponseMapper orderResponseMapper;
+
+    public OrderController(OrderService orderService,
+                           ShoppingCartService shoppingCartService,
+                           UserService userService,
+                           OrderResponseMapper orderResponseMapper) {
+        this.orderService = orderService;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.orderResponseMapper = orderResponseMapper;
+    }
+
+    @PostMapping("/complete")
+    public OrderResponseDto complete(@RequestParam Long userId) {
+        return orderResponseMapper.toDto(orderService
+                .completeOrder(shoppingCartService
+                        .getByUser(userService.get(userId))));
+    }
+
+    @GetMapping("/{userId}")
+    public List<OrderResponseDto> get(@PathVariable Long userId) {
```

Comment:
there is same url schema for both endpoints in task description but you use RequestParam in one case and PathVariable in other